src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -850,7 +873,39 @@ public void integrationTest() throws Exception {
 
     // Start container
     sut.startContainer(id);
-
+    
+    
+    // Copy files to container
+    final String dockerDirectory = Resources.getResource("dockerSslDirectory").getPath();
+    try {
+    	sut.copyToContainer(Paths.get(dockerDirectory), id, "/tmp");
+	} catch (Exception e) {
+		fail("error to copy files to container");
```

Comment:
It's because copying files to a container is only available in docker api >= 1.21 http://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#extract-an-archive-of-files-or-folders-to-a-directory-in-a-container